java-plugin/src/main/java/fr/greencodeinitiative/java/JavaRulesDefinition.java

```diff
@@ -52,7 +52,7 @@ public class JavaRulesDefinition implements RulesDefinition {
     public void define(Context context) {
         NewRepository repository = context.createRepository(REPOSITORY_KEY, LANGUAGE).setName(NAME);
 
-        SonarRuntime sonarRuntime = SonarRuntimeImpl.forSonarQube(Version.create(9, 8), SonarQubeSide.SCANNER, SonarEdition.DEVELOPER);
+        SonarRuntime sonarRuntime = SonarRuntimeImpl.forSonarQube(Version.create(10, 0), SonarQubeSide.SCANNER, SonarEdition.DEVELOPER);
```

Comment:
I don't think you need to create the Sonar runtime manually here. It would also avoid having to change the version at each SonarQube update. I can propose a PR with this specific changes if you want :+1:

Reply:
yes ... why not !? good idea ! please make it for all current languages (PHP, JAVA, PYTHON, JS) and check mobile part to see if it's the same 
create a PR please for these modifications

Reply:
Ok :+1: I will wait for the PR #188 to be integrated before, to avoid conflicts